book/getting-started/system-requirements.rst

```diff
@@ -7,7 +7,7 @@ For successfully running Sulu you'll have to fulfill the following system requir
 
 * Mac OSX, Linux or Windows
 * Webserver (`apache <http://httpd.apache.org/>`_ or `nginx <http://nginx.org/>`_ with enabled URL rewriting)
-* `PHP <http://php.net/>`_ 5.5 or higher
+* `PHP <http://php.net/>`_ 5.5 or higher with php-intl and php-gd
```

Comment:
`php-intl` is probably correct, but you could also use `imagemagick` instead of `php-gd` (what we usually do...). I do not know how we should handle that in here 😕